app/shared/templateTypes.js

```diff
@@ -1,13 +1,20 @@
 const templateTypes = {
-  select: 'select',
-  multiselect: 'multiselect',
-  text: 'text',
-  link: 'link',
-  media: 'media',
+  date: 'date',
+  daterange: 'daterange',
+  geolocation: 'geolocation',
   image: 'image',
+  link: 'link',
   markdown: 'markdown',
-  date: 'date',
+  media: 'media',
+  multidate: 'multidate',
+  multidaterange: 'multidaterange',
+  multiselect: 'multiselect',
+  nested: 'nested',
+  numeric: 'numeric',
+  preview: 'preview',
   relationship: 'relationship',
+  select: 'select',
+  text: 'text',
 };
 
 export { templateTypes };
```

Comment:
nit: why not default?

Reply:
What do you mean by this @bdittes ?

Reply:
the file is called templateTypes and only exports one thing, so why not 'export default templateTypes'?

Reply:
Ooh I see now. Yeah, I could make it a default export.

Reply:
the idea with this was actually stop using export defaults to have naming consistent across files, we had bad experiences with this in the past.

Reply:
Aha! That's fine too. Sam mentioned you might be collecting some of these in a style guide? ;)